adflow/pyADflow.py

```diff
@@ -1448,6 +1449,235 @@ def evalFunctionsSens(self, aeroProblem, funcsSens, evalFuncs=None):
             print("| %-30s: %10.3f sec" % ("Complete Sensitivity Time", finalEvalSensTime - startEvalSensTime))
             print("+--------------------------------------------------+")
 
+    def evalFunctionsSensFwd(self, aeroProblem, funcsSens, evalFuncs=None):
```

Comment:
Is this function something we want to fix and maintain in the code for dot-product tests? 
I am 95% sure this is not working now, will check and add more info here.
Great comments though

Reply:
So in general, I think there would be value in a "direct" implementation to get total derivatives. This however, is not that. It's still adjoint with a more inefficient way to get some of the partials using fwd AD. If you are not willing to implement direct properly, then I say you can remove this. However, there is still value in the explanations and the code etc to guide future implementations of direct. So I am not sure what to do here.

Reply:
@sseraj also looking for your opinion here :pray: I could migrate the comments to a "dev guide" section of the docs?

Reply:
I agree, it would be better in the docs.